finalProject/highLevelCC.md

```diff
@@ -0,0 +1,4 @@
+
+
+when trying to do a shift cypher i think the first thing i need to do is create a variable what stores the alphabet. ```alph = ("ABCDE....XYZ")```
+i will then create a loop that would be ```for (var i = 0; i < alph.length; 1++)``` so it will go through and look to see how long it is. then i would find a way to shift the index of the characters by a certain amount. so when i try to call it like  ```encode("hello", 2);``` jgnnq would appear.
```

Comment:
Good start- you can reference the alphabet array to help you make your letter shifts. You can continue to break all of this into smaller steps, and I encourage you to have a try at coding some of it out!